motion/ruby_motion_query/data.rb

```diff
@@ -64,6 +64,12 @@ def style_name=(value)
       styles[0] = value
     end
 
+    # view.rmq_data.styles = :something
```

Comment:
Good call. This does need to be an array. I'll update with the note and the guard.